src/styleUtils.ts

```diff
@@ -89,10 +89,43 @@ function mergeMarkdownStyleWithDefault(input: PartialMarkdownStyle | undefined):
   return output;
 }
 
+function deepEqualMarkdownStyles(markdownStyle1: MarkdownStyle, markdownStyle2: PartialMarkdownStyle) {
+  const keys1 = Object.keys(markdownStyle1) as Array<keyof MarkdownStyle>;
+  const keys2 = Object.keys(markdownStyle2) as Array<keyof MarkdownStyle>;
+
+  if (keys1.length !== keys2.length) {
+    return false;
+  }
```

Comment:
Since all values of `markdownStyle` are serializable, why can't we just compare `JSON.stringify(markdownStyle1) === JSON.stringify(markdownStyle2)`?

Reply:
Don't we want to treat objects like `{color: '#FF0000', backgroundColor: '#FF0000'}` and `{backgroundColor: '#FF0000', color: '#FF0000'} `as equal? That's the only reason I didn't use `JSON.stringify`, because it maintains order. We could use `JSON.stringify`, but then some logically equal objects(but with changed order of props) would come out as not equal. WDYT?